_includes/metadata.html

```diff
@@ -0,0 +1,142 @@
+<!-- Standard metadata -->
+{%- if site.enable_google_verification -%}
+<meta name="google-site-verification" content="{{ site.google_site_verification }}" />
+{%- endif -%}
+{%- if site.enable_bing_verification -%}
+<meta name="msvalidate.01" content="{{ site.bing_site_verification }}" />
+{%- endif -%}
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<title>
+{%- if site.title == "blank" -%}
+{%- capture title -%}{{ site.first_name }} {{ site.middle_name }} {{ site.last_name }}{%- endcapture -%}
+{%- else -%}
+{%- capture title -%}{{ site.title }}{%- endcapture -%}
+{%- endif -%}
+{%- if page.title != "blank" and page.url != "/" -%}
+{{ title }} | {{ page.title }}
+{%- else -%}
+{{ title }}
+{%- endif -%}
+</title>
+<meta name="author" content="{{ site.first_name }} {{ site.middle_name }} {{ site.last_name }}" />
+<meta name="description" content="{%- if page.description -%}{{ page.description }}{%- else -%}{{ site.description }}{%- endif -%}" />
+{% if page.keywords or site.keywords %}<meta name="keywords" content="{%- if page.keywords -%}{{ page.keywords }}{%- else -%}{{ site.keywords }}{%- endif -%}" />{% endif %}
+<!-- OpenGraph -->
+{%- if site.serve_og_meta -%}
+<meta property="og:site_name" content="{{ site.title }}" />
+<meta property="og:type" content="website" />
+<meta property="og:title" content="{%- if page.title -%}{{ site.title }} | {{ page.title }}{%- else -%}{{ site.title }}{%- endif -%}" />
+<meta property="og:url" content="{{ page.url | prepend: site.baseurl | prepend: site.url | remove_first: 'index.html' }}" />
+<meta property="og:description" content="{%- if page.description -%}{{ page.description }}{%- else -%}{{ site.description }}{%- endif -%}" />
+{% if page.og_image or site.og_image %}<meta property="og:image" content="{%- if page.og_image -%}{{ page.og_image }}{%- else -%}{{ site.og_image }}{%- endif -%}" />{% endif %}
+<meta property="og:locale" content="{{ site.lang }}" />
+<!-- Twitter card -->
+<meta name="twitter:card" content="summary" />
+<meta name="twitter:title" content="{%- if page.title -%}{{ page.title }}{%- else -%}{{ site.title }}{%- endif -%}" />
+<meta name="twitter:description" content="{%- if page.description -%}{{ page.description }}{%- else -%}{{ site.description }}{%- endif -%}" />
+{% if page.og_image or site.og_image %}<meta name="twitter:image" content="{%- if page.og_image -%}{{ page.og_image }}{%- else -%}{{ site.og_image }}{%- endif -%}" />{% endif %}
+{%- if site.twitter_username -%}
+<meta name="twitter:site" content="@{{ site.twitter_username }}" />
+<meta name="twitter:creator" content="@{{ site.twitter_username }}" />
+{%- endif -%}
+{%- endif -%}
```

Comment:
what does this `endif` correspond to?

Reply:
To the OpenGraph condition. More clear in PR https://github.com/alshedivat/al-folio/pull/481/commits/49924aea02900c6405001a875b9df098d3ba9a5d